client/src/components/annotators/VideoAnnotator.vue

```diff
@@ -84,11 +87,39 @@ export default defineComponent({
     }
 
     async function seek(frame: number) {
-      // ref: PTS precision note above
-      video.currentTime = (frame / props.frameRate) + OnePTSTick;
-      data.frame = Math.round(video.currentTime * props.frameRate);
+      /**
+       * requestedTime is the position, in seconds, that was
+       * requested for seek
+       */
+      const requestedTime = frame / props.frameRate;
+      /**
+       * RequestedTrueVideoFrame is the floating point frame number
+       * expected to be found at requested time
+       */
+      const requestedTrueVideoFrame = requestedTime * props.originalFps;
+      /**
+       * trueFrameWidth is the width, in seconds, of a real video frame
+       */
+      const trueFrameWidth = 1 / props.originalFps;
+      /**
+       * lastTrueFrameBoundary is the time, in seconds, of the
+       * previous frame transition boundary
+       */
+      const lastTrueFrameBoundary = (
+        Math.floor(requestedTrueVideoFrame) / props.originalFps
+      );
+
+      if (requestedTime - lastTrueFrameBoundary > (trueFrameWidth / 2)) {
```

Comment:
So I wrote this out because I found it easier for me to follow this way.  The client things there are extra frames because the duration of the video is padded when specifying non whole number frame rates.

That's because:
`trueFrameWidth` = 0.01668335001
`requestedTime` = 5
`requestedTrueVideoFrame` = 299.7  
**RIGHT HERE IS THE PROBLEM** - requesting frame higher than the frames in video.  But the problem is that the video reports it's duration as 5.006 leading you to believe that there should be an additional frame
`lastTrueFrameBoundary` = 299/59.94 = 4.98832165499

Given the above:
`requestedTime - lastTrueFrameBoundary` = 0.01167834501
`trueFrameWidth / 2` = 0.008341675

So the above line is true meaning that we should set the currentTime to:
`data.currentTime`  =  `lastTrueFrameBondary + trueFrameWidth + OnePTSTick` = 5.00501611611
The duration of the video using the frame assumption is 299*1/59.94 = 4.98832165499 
but both ffmpeg and data.duration report the video length as 5.006.  So the client thinks there is another frame to display but there isn't.  So it caps it at the last displayable frame.  This last frame is really 5.006 - 4.98832165499  long or  0.01767834501.  I've looked at videos with 29.97 and other frequencies and there is an offset to the time.  It may not be enough to rollover into thinking there is an additional frame but it could be there.

We may need `nb_frames` from ffprobe as well to know the exact number of frames in the video to accurately position the file.

https://user-images.githubusercontent.com/61746913/125833015-fea6131e-3c33-4a16-be12-7d8092a7a9e4.mp4